src/gpu/intel/ocl/micro_sdpa.cl

```diff
@@ -21,11 +21,14 @@
 #include "gemm_kq.h"
 #include "gemm_vs.h"
 
-/* The quantization parameter may be unique for each token/element */
```

Comment:
Random spot: do we have test cases covering new per-token scaling optimized path and the old path?